cmd/ws/main.go

```diff
@@ -15,7 +15,7 @@ import (
 	chclient "github.com/absmach/callhome/pkg/client"
 	"github.com/absmach/magistrala"
 	grpcChannelsV1 "github.com/absmach/magistrala/internal/grpc/channels/v1"
-	grpcThingsV1 "github.com/absmach/magistrala/internal/grpc/things/v1"
+	grpcClientsV1 "github.com/absmach/magistrala/internal/grpc/things/v1"
```

Comment:
Please change   
`envPrefixClients   = "MG_CLIENTS_AUTH_GRPC_"`   
to    
`envPrefixClients   = "MG_CLIENTS_AUTH_GRPC_"`  
at here  
https://github.com/dborovcanin/magistrala/blob/7a2274d48234c17ba835427efaf49de4dcdddc75/cmd/ws/main.go#L43